examples/cpp/pyperf/PyOffsets.cc

```diff
@@ -194,7 +194,7 @@ extern const struct struct_offsets kPy310OffsetConfig = {
         .ob_type = 8
     },
     .String = {
-        .data = 48,                // offsetof(PyStringObject, ob_sval)
```

Comment:
What object is it, after all?

Reply:
That's not link to any particular object, but location after the `PyASCIIObject`, as explained in header file:
[unicodeobject.h#L69-L71](https://github.com/python/cpython/blob/3.11/Include/cpython/unicodeobject.h#L69-L71).
Verified with gdb, reading cpython code and creating example code.

Reply:
sure